Userland/Libraries/LibWeb/HTML/BrowsingContextContainer.cpp

```diff
@@ -82,8 +82,8 @@ const DOM::Document* BrowsingContextContainer::content_document() const
     VERIFY(m_document);
 
     // 4. If document's origin and container's node document's origin are not same origin-domain, then return null.
-    if (!document->origin().is_same_origin_domain(m_document->origin()))
-        return nullptr;
+    //    if (!document.origin().is_same_origin_domain(this->document().origin()))
+    //        return nullptr;
```

Comment:
What's going on here?

Reply:
:sweat_smile: I forgot to remove this because it didn't use to work without this. Not sure how I didn't catch this, thanks for catching it!